jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/ColumnInfoDetails.kt

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.insights
+
+import com.intellij.ui.SimpleColoredComponent
+import com.intellij.ui.speedSearch.SpeedSearchUtil
+import com.intellij.util.ui.ColumnInfo
+import software.aws.toolkits.jetbrains.utils.ui.setSelectionHighlighting
+import java.awt.Component
+import javax.swing.JTable
+import javax.swing.table.TableCellRenderer
+
+class ColumnInfoDetails(private val fieldName: String) : ColumnInfo<MutableMap<String, String>, String>(fieldName) {
+    private val renderer = FieldColumnRenderer()
+    override fun valueOf(item: MutableMap<String, String>?): String? {
+        if (item != null) {
+            return item[fieldName]
+        }
+        return null
+    }
+    override fun isCellEditable(item: MutableMap<String, String>?): Boolean = false
+    override fun getRenderer(item: MutableMap<String, String>?): TableCellRenderer? = renderer
+}
+
+class FieldColumnRenderer : TableCellRenderer {
```

Comment:
Can you add a comment that this is for applying speed search highlighting? I thought it was not needed at first

Reply:
It is not, made the changes!